src/web/robots.txt

```diff
@@ -0,0 +1,2 @@
+# https://www.robotstxt.org/robotstxt.html
```

Comment:
I think robots.txt should be inside `public` folder`
src/web/public/robots.txt`

https://nextjs.org/blog/next-9-1
```
Starting from Next.js 9.1 we are introducing a new directory called public.

Any files in the public directory will be mapped to the root of the domain.

For example, public/robots.txt will be mapped to /robots.txt.
```

Reply:
Oh you're right! Thanks for the heads up 👍

Reply:
I've moved it to the public folder and I can access the file under http://localhost:8000/robots.txt